src/call/github-events-to-s3.ts

```diff
@@ -21,6 +21,8 @@ export default async function githubEventsToS3(app: Probot, context: any, resour
   const repoName = context.payload.repository?.name;
   const eventName = context.payload.action === undefined ? context.name : `${context.name}.${context.payload.action}`;
 
+  context.uploaded_at = new Date().toISOString();
+
```

Comment:
Is this in the same format inclined with GitHub `created_at`  or `updated_at` ?

Reply:
Can see the format is good in tests, but again the tests just return what we mocked.

Reply:
Right, the format is the same